### DISCUSSION TITLE
Why does deduplication not fill gaps?

### DISCUSSION THREAD

**Discussion by dastrobu** [thanos-io/thanos#4905] [Questions & Answers] [answered]
I have a setup with multiple replicas and deduplication. Some of the replicas don't provided metrics at some points in time as shown below: 
![image](https://user-images.githubusercontent.com/1847260/143550491-63add01e-62c9-40b5-a610-6c4e65ce7f0f.png)

When enabling deduplication, I was expecting the gaps to be filled by combining data from different replicas. But I do still see gaps at some points: 
![image](https://user-images.githubusercontent.com/1847260/143550622-74ac3d53-c53b-4805-a854-81e3ed0af128.png)

I somehow don't seem to have the right understanding of how deduplication works in detail. 
Hints would be helpful on how to analyze the replica data and the deduplicated data. Just looking at some lines with gaps does not explain the current situation very well to me. 

I could potentially increase the scrape interval, but that would also increase storage requirements quite a bit, which I would like to avoid if possible.

**Comment by jmichalek132** [comment, marked as answer]
Known issue https://github.com/thanos-io/thanos/issues/981
